contracts/feeManager/FeeManager.sol

```diff
@@ -122,14 +120,19 @@ contract FeeManager is IFeeManager, Ownable {
      * @inheritdoc IFeeManager
      */
     // slither-disable-next-line reentrancy-events
-    function withdraw(uint256 amount) external override {
-        if (amount > _funds[msg.sender]) {
+    function withdraw(uint256 amount, address beneficiary) external override {
+        if (beneficiary == _feesOwner) {
+            isFinancialOperator(msg.sender);
```

Comment:
This function does not revert, either add a require or substitute with `hasRole`

Reply:
Fixed on the last commit